src/analysisd/decoders/decode-xml.c

```diff
@@ -35,7 +35,7 @@ int getDecoderfromlist(const char *name)
     return (0);
 }
 
-static int addDecoder2list(const char *name)
+static int addDecoder2list(const char *name, char **msg)
```

Comment:
This function could stay as it was originally. LIST_ADD_ERROR occurs when can't allocate memory, Wazuh must print this message in ossec.log